src/containers/Form/FormLayout.tsx

```diff
@@ -86,7 +89,10 @@ export interface FormLayoutProps {
     text?: string;
     status?: boolean;
   };
-  skipCancel?: boolean;
+  errorButtonStatus?: {
```

Comment:
sorryy, this should be errorButtonState
```suggestion
  errorButtonState?: {
```